homeassistant/components/netatmo/__init__.py

```diff
@@ -148,13 +160,30 @@ async def register_webhook(event):
             webhook_register(
                 hass, DOMAIN, "Netatmo", entry.data[CONF_WEBHOOK_ID], handle_webhook
             )
+
+            async def handle_event(event):
+                """Handle webhook events."""
+                if event["data"]["push_type"] == "webhook_activation":
+                    if sub is not None:
+                        _LOGGER.debug("sub called")
+                        sub()
+
+                    if unsub is not None:
+                        _LOGGER.debug("Unsub called")
+                        unsub()
+
+            sub = async_dispatcher_connect(
```

Comment:
The naming is incorrect. This will unsubscribe the dispatcher connect event.

Reply:
Yes, once the webhook is established we don't need to listen for it any more, right?

Reply:
Correct. But what does `sub` mean? Pick names that describe what the variable does.